### PR TITLE
Update ecs using paths function

### PR DESCRIPTION
**Pull request by Hanmac**
Fixes the Failed Test in #49

### CODE REVIEW

**Comment by Hanmac** [comment]
@loevgaard I can't apply `PhpCsFixer\Fixer\ControlStructure\TrailingCommaInMultilineFixer` without breaking php7.4 support

@or-benjamin same there with the mixed type

Is it time to drop php7.4 support?

File: src/Validator/CronExpression.php [hunk 1]
```diff
@@ -17,15 +18,13 @@ class CronExpression extends Constraint
...
      * @param string[]|null $groups
-     * @param mixed $payload
-     * @param array $options
+     * @param mixed|null $payload
      */
     public function __construct(
         string $message = null,
         ?array $groups = null,
-        mixed $payload = null,
+        $payload = null,
```
Comment (Chris53897) [inline src/Validator/CronExpression.php:27]: I am not sure about this change. Are the tests still green with thr orginal code?
mixed $payload = null,

Comment (Hanmac) [reply src/Validator/CronExpression.php:27]: it broke this one:
https://github.com/Setono/CronExpressionBundle/actions/runs/11443626072/job/31836694349

but unit tests seems to be fine?

or is the problem that coding-standard vs StaticCodeAnalyis confict with each other there?

Comment (Chris53897) [reply src/Validator/CronExpression.php:27]: Thanks for the link.
In my opinion, your line should be reverted.

I have created a new Issue.
https://github.com/Setono/CronExpressionBundle/issues/51

Comment (Hanmac) [reply src/Validator/CronExpression.php:27]: Because our tests don't use payload, the tests still pass,
while the StaticCodeAnalyis one does fail

**Comment by loevgaard** [comment]
It would be great to remove PHP 7.4 support and go to 8.1 :)

**Comment by Chris53897** [comment]
https://github.com/Setono/CronExpressionBundle/issues/51

**Comment by Hanmac** [comment]
Closed in favor of #52

**Pull request closed without merging**